pkgs/tools/misc/tremor-rs/default.nix

```diff
@@ -0,0 +1,44 @@
+{ lib, rustPlatform, pkg-config, cmake, llvmPackages, openssl, zlib, gcc, clang
+, fetchFromGitHub, stdenv, installShellFiles }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "tremor";
+  version = "0.10.1";
+
+  src = fetchFromGitHub {
+    owner = "tremor-rs";
+    repo = "tremor-runtime";
+    rev = "v${version}";
+    sha256 = "1z1khxfdj2j0xf7dp0x2cd9kl6r4qicp7kc4p4sdky2yib66512y";
+  };
+
+  cargoSha256 = "sha256-rN/d6BL2d0D0ichQR6v0543Bh/Y2ktz8ExMH50M8B8c=";
+
+  nativeBuildInputs = [ cmake pkg-config clang gcc installShellFiles ];
```

Comment:
Does this really require clang and gcc?

Reply:
It was failing with some weird linker errors without it.

Reply:
Can you link it? This is very uncommon and I have the feeling you miss understood something here.

Reply:
Hmmm. You're right. I just tried it out https://github.com/humancalico/build/runs/1945079089 and it works. I'll make the changes and I'll also bump it to 0.10.2 since it got released a few days ago.

Reply:
Done. Also removed the `zlib` dependency